test/validate.spec.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-undef */
+const { validate } = require('@s1seven/schema-tools-validate');
+const Ajv = require('ajv');
+const addFormats = require('ajv-formats');
+const { readFileSync } = require('fs');
+const { resolve } = require('path');
+const { getRefSchemaUrl } = require('./helpers');
+
+describe('Validate', function () {
+  it('should validate schema', () => {
+    const schemaPath = resolve(__dirname, '../schema.json');
+    const schemaToValidate = JSON.parse(readFileSync(schemaPath, 'utf-8'));
+    const ajv = new Ajv({ strict: false, allErrors: true });
+    addFormats(ajv);
+    //
+    const validateSchema = ajv.compile(schemaToValidate);
+    expect(() => validateSchema()).not.toThrow();
+  });
+
+  const validCertTestSuitesMap = [
+    {
+      certificateName: `valid_certificate_1`,
+    },
+  ];
+
+  validCertTestSuitesMap.forEach(({ certificateName }) => {
+    it(`${certificateName} should be a valid certificate`, async () => {
+      const certificatePath = resolve(__dirname, `./fixtures/${certificateName}.json`);
+      const certificate = JSON.parse(readFileSync(certificatePath, 'utf8'));
+      certificate.RefSchemaUrl = getRefSchemaUrl();
+      //
+      const errors = await validate(certificate);
+      expect(errors).toBeNull();
+    });
+  });
+
+  const invalidCertTestSuitesMap = [
+    {
+      certificateName: `invalid_certificate_1`,
+      expectedErrors: {
```

Comment:
Here we would we need some extra invalid certificates with broken values to ensure our schema is not letting those values passing the validation